### PR TITLE
V1 Implementation

### PR DESCRIPTION
**Pull request by izo0619**

### CODE REVIEW

File: include/temp_converters.h [hunk 1]
```diff
@@ -0,0 +1,75 @@
+#pragma once
+#include <map>
+
```
Comment (eton1234) [inline include/temp_converters.h:3]: I assume this is good

Comment (izo0619) [reply include/temp_converters.h:3]: Just a guard to prevent header files from being included redundantly

File: include/inverter_driver.h [hunk 2]
```diff
@@ -20,39 +18,80 @@ class IInverter
...
+    Inverter(ICAN &can_interface_) : can_interface_(can_interface_){};
+
+    float GetMotorTemperature() override;
+    float GetInverterTemperature() override;
+    float GetRPM() override;
+    void GetStatus() override;
+    // set vals
+    void RequestTorque(float percent) override;
+    void RXCallback();
+    void RequestMotorTemperature(uint8_t freq);
```
Comment (eton1234) [inline include/inverter_driver.h:40]: Might be helpful to make a comment about 1 -> 254 ms, 0 being one time, 255 being stop

**Review by eton1234 — APPROVED** [review]
Idk how the unit conversion stuff works but looks good otherwise

**Pull request merged**